BUILD_AND_INSTALL_FORKLIFT_ON_KIND.md

```diff
@@ -46,6 +46,7 @@ it is possible to install the following source providers as part of k8s kind clu
 - [VMware - vcsim](https://github.com/vmware/govmomi/blob/main/vcsim/README.md)
 - [OpenStack - packstack](https://github.com/kubev2v/packstack-img)
 - [oVirt - fakeovirt and ovirt-imageio](https://github.com/kubev2v/fakeovirt)
+- [OVA - NFS share setup and copying OVA files]()
```

Comment:
in () we can set the ova link.

Reply:
but then pressing that link would start downloading the OVA - it's not that good since the text `OVA - NFS share setup and copying OVA files` doesn't indicate that pressing the link would trigger download

Reply:
a solution for that might be another md file with the link. but not sure if it worth it.

Reply:
yeah, I don't think we have something meaningful to write about that